e2e/pages/session-search.po.ts

```diff
@@ -75,6 +76,36 @@ export class SessionSearchPage {
         await row.element(by.cssContainingText('.clickable', 'Edit')).click()
     }
 
+    async checkIfHasNote(expectedNoteId, expectedNoteValue, ...values: string[]): Promise<boolean> {
+        Logger.log(`Checking if note exists. May fail when multiple notes of given id (${expectedNoteId}) are found`);
+
+        const row = await this.listingRequestsTable.rowThatContainsAtAnyPage(this.listingRequestsTablePaginator, ...values)
+        await browser.wait(ExpectedConditions.visibilityOf(row), Wait.normal, `Listing request with values: ${values} is not visible`)
+
+        Logger.log('Clicking the notes indicator (By looking for the: \'Yes\' button)');
+        await row.element(by.cssContainingText('.clickable', 'Yes')).click()
+
+        let foundNotes: ElementFinder[] = await element(by.id('notesDialog'))
+            .all(by.css('app-note'))
+            .all(by.id(expectedNoteId))
+            .all(by.id('note-textarea'))
```

Comment:
cant we use only this last call to get all notes?

Reply:
Why?